src/Sulu/Bundle/AdminBundle/Metadata/FormMetadata/CacheLifetimeMetadata.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Metadata\FormMetadata;
+
+class CacheLifetimeMetadata
+{
+    private string $type;
+
+    private string $value;
```

Comment:
Is string required due to the xml parsing or what is the reason for not using directly int?

Reply:
The resolver is reponsible to convert the string then into the correct integer value from metadata point it can be something like `* 3 * * *` (type="expression") or `2400` (type="seconds").

Reply:
👍